src/Exceptions/HttpExceptionHtmlResponse.php

```diff
@@ -0,0 +1,50 @@
+<?php
+
+/**
+ * @var string|int $code
+ * @var string $phrase
+ * @var string $href
+ */
+
+if ($code === 0) {
+    $title = $phrase;
+} else if (!empty($phrase)) {
+    $title = "{$code} - {$phrase}";
+}
+
+$phraseHtml = !empty($href) ? "<a href='{$href}'>{$phrase}</a>" : $phrase;
+
+?>
+
+<!DOCTYPE HTML>
+<html lang='en'>
+<head>
+    <title><?= $title ?></title>
+</head>
+<body>
+    <style>
+        * {
+            font-family: Arial, Helvetica, sans-serif;
+            text-align: center;
+        }
+
+        body {
+            background: #1b1c1d;
+            color: white;
+            padding-top: calc(50vh - 95px);
+        }
+
+        body>div {
+            max-width: 90%;
+            margin: auto;
+            width: fit-content;
+        }
+    </style>
+    <div>
+        <h1><?= $code ?> - <?= $phraseHtml ?></h1>
+        <hr>
+        <p><?= $description ?></p>
+    </div>
```

Comment:
```suggestion
<div>
    <h1>
        <?= $code ?>

        <?php if (isset($url)): ?>
            <a href="<?= $url ?>"><?= $title ?></a>
        <?php else: ?>
            <?= $title ?>
        <?php endif ?>
    </h1>
    <hr>
    <p><?= $description ?></p>
</div>
```
_Note, this code WILL NOT work in current state_

The logic for creating `$phraseHtml` both shouldn't be here and is flawed. `$title` is unused _It is also hard for me to understand what actually is supposed to happen here_

**Actual feedback**
- Remove logic from the template
- Treat it as an actual template (aka use formatting like my suggested code, tho it may not be possible for h1 elements, not sure if they render new lines in them)
- Rename `$href` to `$url`, `href` as a property is also used for css inclusion etc.